Find a functionally equivalent code:android.media.MediaCodecInfo.getSupportedTypes:COMMENT
Method Modifier: public      final       
Comment:/**
 * Query the media types supported by the codec.
 */

@@ -1,3 +1,6 @@
 {
-    return MediaCodecList.getSupportedTypes(mIndex);
+    Set<String> typeSet = mCaps.keySet();
+    String[] types = typeSet.toArray(new String[typeSet.size()]);
+    Arrays.sort(types);
+    return types;
 }

